plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -77,51 +81,51 @@ func getTestCases(serialValue int64) []testCase {
 		panic(err)
 	}
 	inet := scalar.Inet{}
-	err = inet.Set("192.168.0.1/24")
-	if err != nil {
-		panic(err)
-	}
-	timeMicrosecond := scalar.Time{
-		Unit: arrow.Microsecond,
-	}
-	err = timeMicrosecond.Set("04:05:06.789000")
-	if err != nil {
-		panic(err)
-	}
-	timeMillisecond := scalar.Time{
-		Int: scalar.Int{
-			BitWidth: 32,
-		},
-		Unit: arrow.Millisecond,
-	}
-	err = timeMillisecond.Set("04:05:06.789")
+	err = inet.Set("192.168.0.0/24")
```

Comment:
We can change this back to `192.168.0.1/24` after https://github.com/cloudquery/plugin-sdk/pull/982 is released